docs/LV_client/tools/mono_logger.md

```diff
@@ -0,0 +1,55 @@
+# mono_logger.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_logger.png"/>
+</p>
+
+### Description
+
+This VI creates text files or binary files depending on the type of sensor.    
```

Comment:
Did you mean for these to be bullet points? If so,

- **Text file:**        
    - State sensor    
    - Ultrasonic    
    - Collision    
    - Radar    
 
- **Binary files:**    
    - IMU    
    - GPS    
    - RPM    
    - LiDAR    
    - Semantic LiDAR    
    - Camera - **Text file:**        
    - State sensor    
    - Ultrasonic    
    - Collision    
    - Radar    
 
- **Binary files:**    
    - IMU    
    - GPS    
    - RPM    
    - LiDAR    
    - Semantic LiDAR    
    - Camera

Reply:
![Screen Shot 2020-09-04 at 4 58 13 PM](https://user-images.githubusercontent.com/41026694/92287592-d2c52d80-eecf-11ea-8c7b-e0f807510694.png)
Don't know why it's not lining up correctly, just needs some tweaking until it shows the right way.